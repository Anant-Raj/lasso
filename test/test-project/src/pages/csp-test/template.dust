{@lasso-page name="page1" packagePath=packagePath /}

<html>
    <head>
        {@lasso-head /}
    </head>
    <body>
        {@lasso-body /}
    </body>
</html>