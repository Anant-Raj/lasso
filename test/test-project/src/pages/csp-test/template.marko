<lasso-page name="page1" package-path="./browser.json"/>
<lasso-resource path="./favicon.ico" var="favicon"/>

<html>
    <head>
        <lasso-head/>
        <link rel="shortcut icon" href="${favicon.url}">
    </head>
    <body>
        <lasso-img src="./marko-logo.png"/>
        <lasso-img src="./marko-logo.png" width="100" height="100"/>
        <script foo="bar" lasso-nonce>console.log('my inline script');</script>
        <lasso-body/>
    </body>
</html>