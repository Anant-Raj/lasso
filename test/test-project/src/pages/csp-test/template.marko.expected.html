<html><head><link rel="stylesheet" href="/static/page1-8b866529.css">
<style nonce="abc">body .inline {
    background-color: red;
}</style><link rel="shortcut icon" href="/static/favicon-c3deb101.ico"></head><body><img src="/static/marko-logo-314d77d4.png" width="240" height="200"><img src="/static/marko-logo-314d77d4.png" width="100" height="100"><script foo="bar" nonce="abc">console.log('my inline script');</script><script src="/static/page1-1097e0f6.js"></script>
<script nonce="abc">console.log('hello-inline');</script></body></html>